### PR TITLE
build: also build libdevmapper in parallel

### PR DESCRIPTION
**Pull request by rchincha**
PR #258 resurrected

Signed-off-by: Ramkumar Chinchani <rchincha@cisco.com>

**What type of PR is this?**

**Which issue does this PR fix**:

**What does this PR do / Why do we need it**:

**If an issue # is not available please add repro steps and logs showing the issue**:

**Testing done on this change**:

**Automation added to e2e**:

**Will this break upgrades or downgrades?**

**Does this PR introduce any user-facing change?**:

```release-note

```

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: build.yaml [hunk 1]
```diff
@@ -39,7 +39,7 @@ build-env:
     tar -xf /stacker/v2_03_15.tar.gz
     cd lvm2-*
     ./configure --enable-static_link
-    make install_device-mapper
+    make -j$(grep -c processor /proc/cpuinfo) install_device-mapper
```
Comment (smoser) [inline build.yaml:42]: fwiw, this just silently-ish swallows failure of the 'grep' command.

better would be:

```sh
numcpu=$(grep -c processor /proc/cpuinfo)
make -j${numcpu} install_device-mapper
```

example:

```
$ cat /tmp/foo
#!/bin/sh
set -e
echo make -j$(bogus command) foo
echo still here

$ sh -x /tmp/foo
+ set -e
+ bogus command
/tmp/foo: 3: bogus: not found
+ echo make -j foo
make -j foo
+ echo still here
still here
```

you end up calling make like `make -j install_device-mapper`.  make will probably fail, but its just odd.

**Review by smoser — CHANGES_REQUESTED** [review]

File: build.yaml (hunk 1 above)
Comment (rchincha) [reply build.yaml:42]: fixed

**Comment by rchincha** [comment]
/usr/lib/go/pkg/tool/linux_amd64/link: running gcc failed: exit status 1
/usr/lib/gcc/x86_64-alpine-linux-musl/12.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find -ldevmapper: No such file or directory
collect2: error: ld ret


Not sure devmapper code can handle parallel make

**Comment by rchincha** [comment]
Closing this PR for now.

**Pull request closed without merging**